ttn/application.py

```diff
@@ -23,11 +23,14 @@
 else:
     os.environ["GRPC_SSL_CIPHER_SUITES"] = stubs.MODERN_CIPHER
 
+TIME_OUT = 60
+
 
 class ApplicationClient:
 
     def __init__(self, app_id, token_or_key, **kwargs):
         self.app_id = app_id
+        self.discovery_address = "discovery.thethings.network:1900"
```

Comment:
Why are you using `kwargs` for named parameters? Using named parameters is clearer and more powerful. e.g. Python IDEs, like PyCharm, can parse function signatures to detect named parameters, but they can't if you're using `kwargs`. `kwargs` should be reserved for non-predictible named parameters.